src/main/kotlin/com/sourcegraph/cody/chat/ui/ChatPanel.kt

```diff
@@ -122,8 +122,8 @@ class ChatPanel(project: Project, chatSession: ChatSession) :
   ) {
     if (messagesPanel.componentCount == 1) {
       modelDropdown.isEnabled = false
+      promptPanel.updateEmptyTextAfterFirstMessage()
```

Comment:
Why that change?
Name of the function does not reflect what it does anymore.

Reply:
why? `componentCount == 1` when we have only the hello message

Reply:
I'm blind. You actually fixed it, I somehow thought it was other way around.